nearlib/nearclient.js

```diff
@@ -37,22 +37,17 @@ class NearClient {
             args = {};
         }
         const serializedArgs = Buffer.from(JSON.stringify(args)).toString('hex');
-        const result = await this.jsonRpcRequest('abci_query', [`call/${contractAccountId}/${methodName}`, serializedArgs, '0', false]);
-        const response = result.response;
-        let logs = [];
-        if (response.log !== undefined && response.log.length > 0) {
-            logs = response.log.split('\n');
-        }
-        logs.forEach(line => {
-            console.log(`[${contractAccountId}]: ${line}`);
-        });
-        // If error, raise exception after printing logs.
-        const code = response.code || 0;
-        if (code != 0) {
-            throw Error(response.info);
+        const ignoreErrors = true;
```

Comment:
Could you leave a comment somewhere about what this does and why we need this?

Reply:
If you mean `ignoreErrors` it just needs to be removed, accidental leftover